### PR TITLE
Lg 44

### PR DESCRIPTION
**Pull request by TaylorAkin**
LG-44 Fixes warnings and errors from status report
 - Updates CKEditor plugins
 - Updates htaccess file for public files
 - uncomments settings.php solr configuration
 - updates twig vardumper
 - Adds update hook for entity updates needed for taxonomy terms and paragraphs
 - Removes deprectaed settings.php line

### CODE REVIEW

**Review by mattsmith14 — APPROVED** [review]
It is always a little hard to find what actually needs reviewed with this many files. But the settings and update hook look good to me!

File: config/sync/search_api.index.pantheon_index.yml [hunk 1]
```diff
@@ -296,4 +296,4 @@ options:
   cron_limit: 50
   index_directly: true
   track_changes_in_references: true
-server: null
+server: pantheon
```
Comment (chris-allen) [inline config/sync/search_api.index.pantheon_index.yml:299]: This should be the job of `settings.php`

File: config/sync/search_api.server.pantheon.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 uuid: 803e77cc-6370-4f97-b41f-81e97ef33099
 langcode: en
-status: false
+status: true
```
Comment (chris-allen) [inline config/sync/search_api.server.pantheon.yml:3]: Not what you want

File: modules/custom/lex_custom/lex_custom.module [hunk 3]
```diff
@@ -26,4 +26,25 @@ function lex_custom_system_breadcrumb_alter(Breadcrumb &$breadcrumb, RouteMatchI
...
     }
   }
-}
\ No newline at end of file
+}
+
+/**
+ * Implements hook_update_N().
+ */
+function lex_custom_update_9401() {
```
Comment (chris-allen) [inline modules/custom/lex_custom/lex_custom.module:34]: Give credit where you got this. I'm curious to read.

**Review by chris-allen — CHANGES_REQUESTED** [review]

**Comment by chris-allen** [comment]
@TaylorAkin I pushed a commit to update .gitattributes in hopes of removing some noise in the PR diff.

File: modules/custom/lex_custom/lex_custom.module (hunk 3 above)
Comment (TaylorAkin) [reply modules/custom/lex_custom/lex_custom.module:34]: Yea I had it in one of the links I tagged you and Matt on. I can put it here too.

**Review by chris-allen — APPROVED** [review]

**Pull request merged**